bindinfo/handle.go

```diff
@@ -485,10 +491,10 @@ func (h *BindHandle) SetBindRecordStatus(originalSQL string, binding *Binding, n
 	updateTsStr := updateTs.String()
 
 	if binding == nil {
-		_, err = exec.ExecuteInternal(context.TODO(), `UPDATE mysql.bind_info SET status = %?, update_time = %? WHERE original_sql = %? AND update_time < %? AND status IN (%?, %?)`,
+		_, err = exec.ExecuteInternal(ctx, `UPDATE mysql.bind_info SET status = %?, update_time = %? WHERE original_sql = %? AND update_time < %? AND status IN (%?, %?)`,
```

Comment:
If the source label is requested to be filled before raising a new execution, we may need to find a way to force this constraint for example adding checks at the bottom. Others may be aware of this if he's going to add an internal job or a inner worker.

Reply:
The case may be similar for `RunInNewTxn`. If we don't want to miss setting the label for `RunInNewTxn`, is it necessary to add the label as an parameter of `RunInNewTxn`?

An alternative is to check whether the label exists and print a log if it does not. It may be a good idea if we don't want to break things unexpectedly.

Reply:
It'll panic in test mode, and will log warnings in production mode now.